tests/data/rawvoice_mycast.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Namespace Test",
+  "episodes": [],
+  "mycast": "yes"
```

Comment:
Could be normalized to `true`/`false` or removed completely, as we don't really support any MyCast sign-in?

<q> mycast, if present as a sub-item of <channel>, indicates whether the specific program is part of the currently signed in user’s MyCast subscription.</q>